pkg/macchanger

```diff
@@ -0,0 +1,31 @@
+[mirrors]
+https://github.com/alobbs/macchanger/archive/1.7.0.tar.gz
+
+[vars]
+filesize=306344
+sha512=55fbfe4f4e94f0822e7a71b8c7052ed6c3a05066859044498bd68d068c33902b5718e92c8e00b73af71b57ce825644e1f81eb4dd7ef11701f197b953d7853aff
+tarball=macchanger-1.7.0.tar.gz
+pkgver=1
+
+[deps]
+
+[build]
+# http://stackoverflow.com/questions/6381526/what-is-the-significance-of-caddr-t-and-when-is-it-used
+# https://github.com/alobbs/macchanger/issues/14
+sed -i 's/(caddr_t)/(void *)/' src/netinfo.c
```

Comment:
$ find /opt/musl/include/ | xargs grep caddr
/opt/musl/include/sys/types.h:typedef char *caddr_t;

in that file we find
`#if defined(_GNU_SOURCE) || defined(_BSD_SOURCE)
...
typedef char *caddr_t;
``
that means: we need to #include <sys/types.h>
and eventually #define _GNU_SOURCE or _BSD_SOURCE but the latter is activated by sys/features.h by default so it's only necessary if the default feature test macros have been overridden explicitly)
```